README.md

```diff
@@ -10,30 +10,51 @@ Expected environment:
   - Node Feature Discovery (NFD) Operator
   - Kernel Module Management (KMM) Operator
 
-### SFC MachineConfig
+### SFC out of tree driver
 
-To apply the SFC MachineConfig, it is a prerequisite that there exists a local docker registry containing an image with the SFC module built and present in /opt in the node filesystem. This is expected to be present at the following location:
+The recommended way of installing onload's sfc driver is using [KMM](https://github.com/rh-ecosystem-edge/kernel-module-management/)
 
+#### Deploy using KMM
+
+Before you apply the `Module` custom resource for the SFC driver you must
+remove the existing in-tree driver.
+
+For each appropriate node in the cluster:
+```console
+# rmmod sfc
 ```
-image-registry.openshift-image-registry.svc:5000/openshift-kmm/sfc-module
+
+Then apply the manifest:
+
+```console
+$ oc apply -f sfc/kmm/sfc-module.yaml
 ```
 
-To generate this image, please use SFC module:
+#### Day-0 deployment using MachineConfig
+
+To apply the SFC MachineConfig, it is a prerequisite that there exists a local
+docker registry containing an image with the SFC module built and present in
+`/opt` in the node filesystem. This is expected to be present at the following
+location:
```

Comment:
Is the expected user flow detailed? eg. Deploy module manifest to an existing non-sfc-house-network development cluster to perform this bootstrapping step.